lib/utils.js

```diff
@@ -1,67 +1,12 @@
-const url = require('url');
 const crypto = require('crypto');
-const { ipCheck, errors } = require('arsenal');
+const { errors } = require('arsenal');
 
 const { config } = require('./Config');
 const constants = require('../constants');
 
 const utils = {};
 
 /**
- * Get bucket name from the request of a virtually hosted bucket
```

Comment:
How did you decide what to leave in this file? Is there any unifying theme to the remaining functions? Should we put them somewhere else?